datahub-web-react/src/app/onboarding/OnboardingTour.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button } from 'antd';
+import React, { useContext, useEffect, useState } from 'react';
+import Tour from 'reactour';
+import { useBatchUpdateStepStatesMutation } from '../../graphql/step.generated';
+import { EducationStepIdsAllowlistContext } from '../../providers/EducationStepIdsAllowlistContext';
+import { EducationStepsContext } from '../../providers/EducationStepsContext';
+import { StepStateResult } from '../../types.generated';
+import { useGetAuthenticatedUser } from '../useGetAuthenticatedUser';
+import { convertStepId, getStepsToRender } from './utils';
+
+type Props = {
+    stepIds: string[];
+};
+
+export const OnboardingTour = ({ stepIds }: Props) => {
+    const { educationSteps, setEducationSteps } = useContext(EducationStepsContext);
+    const { educationStepIdsAllowlist } = useContext(EducationStepIdsAllowlistContext);
+    const userUrn = useGetAuthenticatedUser()?.corpUser.urn;
+    const [isOpen, setIsOpen] = useState(true);
+    const [reshow, setReshow] = useState(false);
+    const accentColor = '#5cb7b7';
+
+    useEffect(() => {
+        function handleKeyDown(e) {
+            // Allow reshow if Cmnd + Ctrl + T is pressed
+            if (e.metaKey && e.ctrlKey && e.key === 't') {
+                setReshow(true);
+                setIsOpen(true);
+            }
+        }
+        document.addEventListener('keydown', handleKeyDown);
+    }, []);
+
+    const steps = getStepsToRender(educationSteps, stepIds, userUrn || '', reshow);
+    const filteredSteps = steps.filter((step) => step.id && educationStepIdsAllowlist.includes(step.id));
+    const filteredStepIds: string[] = filteredSteps.map((step) => step?.id).filter((stepId) => !!stepId) as string[];
+
+    const [batchUpdateStepStates] = useBatchUpdateStepStatesMutation();
+
+    function closeTour() {
+        setIsOpen(false);
+        setReshow(false);
+        const convertedIds = filteredStepIds.map((id) => convertStepId(id, userUrn || ''));
+        const stepStates = convertedIds.map((id) => ({ id, properties: [] }));
+        batchUpdateStepStates({ variables: { input: { states: stepStates } } }).then(() => {
+            const results = convertedIds.map((id) => ({ id, properties: [{}] } as StepStateResult));
+            setEducationSteps((existingSteps) => (existingSteps ? [...existingSteps, ...results] : results));
+        });
+    }
+
+    if (!filteredSteps.length) return null;
+
+    return (
+        <Tour
+            onRequestClose={closeTour}
+            steps={filteredSteps}
+            isOpen={isOpen}
+            scrollOffset={-100}
```

Comment:
what does this do? just curious

Reply:
Per the docs 
![image](https://user-images.githubusercontent.com/19418814/205784881-bfe7a83c-dd33-4273-8cb3-4ea83307cfd4.png)